zeton/templates/caregiver_panel.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'index_child.html' %}
+{% block add_points %}
+<center>
```

Comment:
Przy okazji można by pozbyć się bloku `<center>`, gdyż jest on wycofywany: https://developer.mozilla.org/pl/docs/Web/HTML/Element/center

Reply:
można zamiast tego wrzucić `<div>` z klasą `centered_block`, która jest odpowiednio ostylowana

Reply:
- usunąłem tagi <center> z powyższych szablonów
- dodałem klasę text-center do poszczególnych elementów, tak żeby to jakoś wyglądało. 

Dalsze zmiany wizualne pozostawiam chętnym.